electron-builder.yml

```diff
@@ -29,7 +35,7 @@ files:
   - '!node_modules/mammoth/{mammoth.browser.js,mammoth.browser.min.js}'
 asarUnpack:
   - resources/**
-  - '**/*.{node,dll,metal,exp,lib}'
```

Comment:
Electron-builder now automatically detects files ending in node/dll and sets them as asarUnpack.